src/platform/plugins/shared/data/server/search/session/session_service.ts

```diff
@@ -301,6 +302,8 @@ export class SearchSessionService implements ISearchSessionService {
     const searchInfo: SearchSessionRequestInfo = {
       id: searchId,
       strategy,
+      status: SearchStatus.IN_PROGRESS,
```

Comment:
Can we update `trackId` to include the entire response (not just the ID) and infer this from the response instead of hardcoding to `IN_PROGRESS`?